shaun-lab26/src/component/category-item/index.js

```diff
@@ -0,0 +1,5 @@
+import React from 'react'
```

Comment:
this component should be displaying the category's name and budget, a delete button onClick the category is removed from app state, and should display a category form which onComplete should update the component